packages/opentelemetry-types/src/trace/span.ts

```diff
@@ -106,6 +106,11 @@ export interface Span {
    */
   end(endTime?: TimeInput): void;
 
+  /**
+   * Returns the flag whether this span is ended.
+   */
+  isEnded(): boolean;
```

Comment:
This is not defined in the API specs, I think it is safer to add at SDK level. If you really want this to be part of the API, please open an issue on specs first.

Reply:
Is it ok to add it to `ReadableSpan` ? I could do `span.toReadableSpan().isSpanEnded()`, otherwise I could use `endTime` but I think that It's better to use isSpanEnded()

WDYT ?

Reply:
I find a way to handle this in HttpPlugin only without impacting the sdk/spec. So we stay with `types.Span`.